### ISSUE TITLE
PodeWebSteps steps skip and limit deplicate execute. and output.

### ISSUE THREAD

**Issue by ili101** [Badgerati/Pode.Web#390]
Hi
Trying to make a "tool" with multiple steps, I used PodeWebSteps as it looks like the most suitable, it works but have some UI problems I need help with (I put it all here for now, tell me if you want me to split it into separate issues).

## Steps can be accidentally skipped.
Not sure if it's only from accidental mouse double click or can occur in other ways. 
1. Load the code.
2. On the first step click "Next" 2 or more times.
#### Behavor
You will usually end up in the third step (even though step 2 should be unpassable in this example)
#### Expected behavior
Option 1: Step 1 executed multiple times and you end up on Step 2.
Option 2: Step 1 once, other clicks ignored and you end up on Step 2 (see next feature request).

## Prevent duplicate execution of steps
Similar to above accidental mouse double click execute the same steps multiple times.
Is it possible to add an option to disable the Next/Submit buttons while the script is running?
Like in addition to the already implemented spinning indicator on the Next/Submit buttons while the script is running, disable the button.
Maybe could be done with `Register-PodeWebEvent`?

## Output with `Out-PodeWebTextbox`
1. Uncomment the `Out-PodeWebTextbox` line and load the code.
2. Click the first step.
3. Additionally: go back and foreword and so on.
#### Behavor
Textbox added at the top, going back it is at the same line as the "Next".
#### Expected behavior
It is put below the other stuff, and stay there on Next/Back/Submit similar to how `Out-PodeWebTextbox` work on a button for example.

## Output with `New-PodeWebTextbox -NoForm`
As a workaround to the `Out-PodeWebTextbox` I added a text box with `New/Update-PodeWebTextbox`.
1. Uncomment the `-NoForm` line and load the code.
#### Behavor
Textbox have narrow width, adding `-Width` doesn't help.
#### Expected behavior
The text box will take all the width similar to how `Out-PodeWebTextbox` show.

### Code
```powershell
Start-PodeServer -Threads 3 {
    Add-PodeEndpoint -Address localhost -Port 8092 -Protocol Http
    Enable-PodeSessionMiddleware -Secret 'ZZZ' -Duration (60 * 60 * 24) -Extend

    Use-PodeWebTemplates -Title 'Tool'

    $WizardSection = New-PodeWebAccordion -Bellows @(
        New-PodeWebBellow -Name 'wizard' -Content @(
            New-PodeWebSteps -Name 'Steps' -Steps @(
                New-PodeWebStep -Name 'Step1' -Icon 'cog-outline' -Content @(
                    New-PodeWebTextbox -Name 'Param 1'
                ) -ScriptBlock {
                    $WebEvent.Session.Data.SubscriptionInfo += "`nStep1"
                    $WebEvent.Session.Data.SubscriptionInfo | Update-PodeWebTextbox -Id 'SubscriptionInfoTextBox' -Multiline
                    # $WebEvent.Session.Data.SubscriptionInfo | Out-PodeWebTextbox -Multiline -Preformat
                    Start-Sleep 5
                }
                New-PodeWebStep -Name 'Step2' -Icon 'cog-outline' -Content @(
                    New-PodeWebTextbox -Name 'Subscription ID'
                ) -ScriptBlock {
                    $WebEvent.Session.Data.SubscriptionInfo += "`nStep2"
                    $WebEvent.Session.Data.SubscriptionInfo  | Update-PodeWebTextbox -Id 'SubscriptionInfoTextBox' -Multiline
                    Out-PodeWebValidation -Name 'Subscription ID' -Message 'Should not be passible to go to next step "Step3"'
                }
                New-PodeWebStep -Name 'Submit' -Icon 'cog-outline'
            ) -ScriptBlock {
                $WebEvent.Session.Data.SubscriptionInfo += "`nEnd"
                $WebEvent.Session.Data.SubscriptionInfo  | Update-PodeWebTextbox -Id 'SubscriptionInfoTextBox' -Multiline
            }
        )
    )
    $InfoSection = New-PodeWebCard -Name 'SubscriptionInfo' -NoTitle -Content @(
        New-PodeWebButton -Name Reset -Icon 'refresh' -ScriptBlock {
            $WebEvent.Session.Data.SubscriptionInfo = ''
            $WebEvent.Session.Data.SubscriptionInfo  | Update-PodeWebTextbox -Id 'SubscriptionInfoTextBox' -Multiline
        }
        New-PodeWebContainer -NoBackground -Content @(
            New-PodeWebTextbox -Name 'Subscription' -Multiline -Preformat -ReadOnly -Id 'SubscriptionInfoTextBox' #-NoForm
        )
    )
    Add-PodeWebPage -Name 'Page1' -Icon 'cart-minus' -Layouts $WizardSection, $InfoSection
}
```

### Platform
 - OS: Linux, Windows
 - Browser: Firefox
 - Versions:
   - Pode: 2.7.2
   - Pode.Web: Pode.Web v0.8.2
   - PowerShell: 7.3.1

**Comment by Badgerati** [comment]
Hey @ili101,

I'll have to investigate the `Out-PodeWebTextbox` one more when I get chance, but for the duplicate clicking of Next/Back/Submit I agree we should just disable the button via JavaScript and re-enable it once it's finished processing the script - and that we could do the same for Buttons as well, likely even Forms, etc.

I'm just catching up with items on Pode.Web, so I'll get round to the PRs soon 😄 

Thanks!

**Comment by ili101** [comment]
Thank you,
Sounds good, I managed to disable buttons by adding `disable(btn);` / `enable(btn);` default.js but not for steps, hopefully you can do it better.
I also manage to replicate the other source of duplicate execution I was suspecting seeing sometimes:
In "Step 1", while being in "Param 1" textbox hit the Enter key. step 1 will post twice. (Looks like it happens only on the first step and only with the keyboard)

**Comment by Badgerati** [comment]
This was resolved by #533, I just forgot to close :)
